### PR TITLE
Create k3z0.json

### PR DESCRIPTION
**Pull request by k3z0**
Learning GitHub, adding my own pin.

### CODE REVIEW

**Comment by davidnhz** [comment]
👍

**Comment by k3z0** [comment]
ok

File: _pins/k3z0.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: "k3z0"
```
Comment (crichID) [inline _pins/k3z0.json]: @k3z0 you will need to add three dashes to the beginning and end of the file `---`

Comment (crichID) [inline _pins/k3z0.json]: @k3z0 you should remove the quotes around your username!

**Comment by brianamarie** [comment]
@k3z0 :wave: Let us know if you want or need any help making these changes. If in a few days the changes still aren't made, one of the GitHub trainers may make the changes for you to keep the repository clean. ✨

**Pull request merged**